crates/driver/src/infra/config/file/mod.rs

```diff
@@ -180,6 +180,10 @@ struct SolverConfig {
     /// Timeout configuration for the solver.
     #[serde(default, flatten)]
     timeouts: Timeouts,
+
+    /// Datetime when the CIP38 should be activated.
+    #[serde(default)]
+    cip38_activation: Option<chrono::DateTime<chrono::Utc>>,
```

Comment:
Is this value only optional for sth like testing purposes (i.e. it's annoying having to add that value)?
Or do we foresee a scenario where we would have to revert back to the old scoring mechanism without breaking any potential solver accounting infra?